modules/structs/repo.go

```diff
@@ -278,6 +278,16 @@ type CreateBranchRepoOption struct {
 	OldRefName string `json:"old_ref_name" binding:"GitRefName;MaxSize(100)"`
 }
 
+// UpdateBranchRepoOption options when updating a branch in a repository
+// swagger:model
+type UpdateBranchRepoOption struct {
+	// New branch name
+	//
+	// required: true
+	// unique: true
+	Name string `json:"name" binding:"Required;GitRefName;MaxSize(100)"`
```

Comment:
Unrelated to this PR. But why is the max size 100 for every API request? Maybe it should be 255.

Reply:
![image](https://github.com/user-attachments/assets/d4405e68-95ab-4821-a47c-89205aed1d8c)
It is limited to 100 in frontend. So 100 is correct.